src/scripts/constants.ts

```diff
@@ -339,10 +352,14 @@ export module Constants {
 	}
 
 	export module Settings {
-		export var timeBetweenTooltips = 1000 * 60 * 60 * 24 * 7 * 3; // 21 days
-		export var noOpTrackerTimeoutDuration = 20 * 1000; // 20 seconds
+		export var fontSizeStep = 2;
+		export var maxClipSuccessForRatingsPrompt = 12; // TODO waiting for consensus on this value
+		export var maximumJSTimeValue = 8640000000000000; // http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
```

Comment:
Eh I guess it doesn't matter since its a max time value.

100000000 * 24 * 60 * 60 * 1000; // 100K days in milliseconds